src/helix-ui/elements/HXRadioSetElement.js

```diff
@@ -0,0 +1,54 @@
+import { HXElement } from './HXElement';
+import { STATE } from './HXFormControlElement';
+
+/*
+ * <hx-radio-set
+ *  @hxchange.stop
+ *  @hxdirty.stop
+ *  @hxtouch.stop
+ * >
+ * </hx-radio-set>
+ */
+
+/**
+  * Defines behavior for the `<hx-radio-set>` element.
+  *
+  * @extends HXFormControlElement
+  * @hideconstructor
+  * @since 0.16.0
+  */
+export class HXRadioSetElement extends HXElement {
+    static get is () {
+        return 'hx-radio-set';
+    }
+
+    $onConnect () {
+        this.addEventListener('hxchanged', this._onHxChanged);
+        this.addEventListener('hxdirty', this._onHxDirty);
+        this.addEventListener('hxtouched', this._onHxTouched);
+    }
+
+    $onDisconnect () {
+        this.removeEventListener('hxchanged', this._onHxChanged);
+        this.removeEventListener('hxdirty', this._onHxDirty);
+        this.removeEventListener('hxtouched', this._onHxTouched);
+    }
+
+    /** @private */
+    _onHxChanged (evt) {
+        evt.stopPropagation();
+        this.$defaultAttribute(STATE.changed, '');
+    }
+
+    /** @private */
+    _onHxDirty (evt) {
+        evt.stopPropagation();
+        this.$defaultAttribute(STATE.dirty, '');
```

Comment:
While this is functional, we should look into refactoring the state logic in Phase 3. Please add a 

```
// TODO: revisit logic in phase 3
```

Reply:
comment added to revisit/refactor in Phase III